🔧-How-to-Install.md

```diff
@@ -1,9 +1,14 @@
 ### 🐳 Docker
 
 ```bash
+curl -sSL https://get.docker.com/ | CHANNEL=stable sh
+apt update && apt install docker-compose-plugin
 docker run -d --restart=always -p 3001:3001 -v uptime-kuma:/app/data --name uptime-kuma louislam/uptime-kuma:1
 ```
-
+Check if its running: 
+```bash
+docker container ls
+```
```

Comment:
Please include the expected result. (I can't comment it because of limitations with suggesting code-blocks)
Currently, I don't see the point of adding this check, but with an expected result I would see the benefit (or can `docker run` fail in non-obvious ways?).